test/functional/helpFn.ts

```diff
@@ -353,8 +353,12 @@ export const getDynamicDispatchExtrinsicParams = async (scheduleType: string) =>
  * getContext: Get test context
  * @returns context object: { scheduler, observer, keyringPair }
  */
-export const getContext = async () => ({
-  scheduler: new Scheduler(OakChains.STUR),
-  observer: new Observer(OakChains.STUR),
-  keyringPair: await getKeyringPair(),
-});
+export const getContext = async () => {
+  const chain = OakChains.STUR;
+  const providerUrl = process.env.PROVIDER_URL;
```

Comment:
Do it here,
`const providerUrl = providerUrl || OakChainWebsockets[chain]`

or even better, in the beginning of index.tx, set the value of `process.env.PROVIDER_URL` to the default value `OakChainWebsockets[chain]` if it is undefined.